spec/features/admin_categories_spec.rb

```diff
@@ -34,4 +34,16 @@
     expect(page).to have_content('Category 5')
     expect(current_path).to eq(admin_categories_path)
   end
+
+
+  it 'deletes a category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.